.github/workflows/test.yml

```diff
@@ -10,14 +10,14 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Setup Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 14
           cache: 'yarn'
           cache-dependency-path: yarn.lock
 
       - name: Install Dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --immutable --mode=skip-build
```

Comment:
`--frozen-lockfile` is an outdated alias of `--immutable` - if you search for `--frozen-lockfile` you'll end up on an old version of the docs for yarn and have a list of cli flags that no longer work.